maker/src/actor_system.rs

```diff
@@ -116,6 +117,11 @@ where
             &oracle_addr,
         )));
 
+        let libp2p_collab_settlement_addr =
+            close_position::maker::Actor::new(executor.clone(), n_payouts)
+                .create(None)
+                .spawn(&mut tasks);
```

Comment:
❓ Shall we follow the new pattern of supervising permanent actors?

Reply:
I think we should - and also consider this for the new `rollover` actor.

Reply:
OK, I'll add supervision for both of them into this PR.